cv19ResSupportV3/V3/Boundary/Requests/CreateHelpRequestCallRequest.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace cv19ResSupportV3.V3.Boundary.Requests
+{
+    public class CreateHelpRequestCallRequest
+    {
+        public int HelpRequestId { get; set; }
```

Comment:
If we don't want the user of the API to be able to provide an Id, we can simply exclude it from the Presentation layer boundary. The model binder will simply not map that field from JSON onto this object (as this object does not have this field).

Reply:
In this case C# binds the fields from the object in the request body to the CreateHelpRequestCallRequest object within a controller. It matches the fields of the 2 objects by their names. If it stumbles upon a field in a request body object who's name is not among the names of the object in the controller, then it ignores that field.

I see that the CreateHelpRequestCallRequest object contains a field "HelpRequestId". In containing this field, this object  allows for accepting a "HelpRequestId" value from the request. Since we don't want the user to be able to provide and hence control the Id of a created object, we can instead remove the "HelpRequestId" field from this C# object. Otherwise we end up carrying around and mapping (factory) the field that we don't want to deal with.

Though I'm not suggesting to do the changes for this now - I agree that this PR is already massive.

Reply:
The `HelpRequestId` is actually the ID of the Help Request that the Call should be assigned to - rather than the ID of the new entity to be created.

I'd argue that the HelpRequestID should also not be in the body, as it's already referenced in the URI to the endpoint.

Reply:
Oh, I see. So it's fine then... I agree it shouldn't be provided via body if it's provided via url... but I think we can keep it as is so long we overwrite it with the value from the url as the 1st thing todo in the controller. I say it's alright if we keep it for later as this PR had enough delay already :thinking: